src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -54,7 +54,15 @@ See the accompanying LICENSE file for applicable license.
     
     <basename property="dita.input.filename" file="${args.input}" />
     <basename property="dita.map.filename.root" file="${dita.input.filename}" suffix=".ditamap" />
-    <basename property="dita.topic.filename.root" file="${dita.input.filename}" suffix=".dita" />
+    <condition property="dita.topic.filename.extension" value=".xml">
```

Comment:
OK, yes, nobody is going to use a `.beep` extension. In the real world, I've only ever encountered `.xml` and `.dita`. But we should continue to allow other extensions. Also - I have encountered maps that included `.xml` files from one source and `.dita` files from another, so hard coding a single extension would not work for those.